controllers/volumereplicationgroup_controller_test.go

```diff
@@ -859,6 +905,120 @@ func (v *vrgTest) waitForVRCountToMatch(vrCount int) {
 		vrCount, v.vrgName, v.namespace)
 }
 
+func (v *vrgTest) updateVRGSpec(vrgName string, state ramendrv1alpha1.ReplicationState) {
+	retries := 0
+	maxRetries := 2
+
+	for retries < maxRetries {
+		vrg := v.getVRG(vrgName)          // get most recent copy of VRG
+		vrg.Spec.ReplicationState = state // requires an extra step; will update along with status
+
+		err := k8sClient.Update(context.TODO(), vrg)
+
+		if err != nil {
+			if errors.IsConflict(err) {
+				retries++
+
+				continue
+			} else {
+				Expect(err).NotTo(HaveOccurred(), "failed to update the spec of vrg %s", vrg.Name)
+			}
+		} else {
```

Comment:
Do we need this else condition? Because the function does not return any particular data type and this else condition just returns and there is no code after this else condition.

Reply:
Seems reasonable. This has been removed